doc/classes/ReflectionProbe.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Capture its surroundings as a dual parabolid image, and stores versions of it with increasing levels of blur to simulate different material roughnesses.
+		The [ReflectionProbe] is used to create high-quality reflections at the cost of performance. It can be combined with [GIProbe]s and Screen Space Reflections to achieve high quality reflections. ReflectionProbes render all objects within their [member cull_mask], so updating them can be quite expensive. It is best to update them once with the important static objects and then leave them.
```

Comment:
Same as GIProbe, use `[ReflectionProbe]` when talking about the Node, or "reflection probes" when talking about probes used for reflection.

Reply:
In practice it's fine already in this class, just in this paragraph there's a stray "ReflectionProbes".